src/ui/public/timepicker/kbn_global_timepicker.html

```diff
@@ -28,5 +31,6 @@
       <i aria-hidden="true" class="fa fa-clock-o"></i>
       <pretty-duration from="timefilter.time.from" to="timefilter.time.to"></pretty-duration>
     </button>
+    <i ng-click="foward()" class="time-nav-btn fa fa-arrow-right ng-scope"></i>
   </li>
 </ul>
```

Comment:
I think that, ultimately, we should rebuild the timepicker component so that these arrow buttons are part of the dropdown that appears when you click the time range. I don't think the user needs to have them visible at all times. However, this will take some time to implement, and I don't think we should hold up this PR for that.

I think the immediate problem is that the "back" arrow blends in with the + and - icons. We can create the separation you mention by moving them both to the right side, and letting the time range itself create visual separation:

![image](https://cloud.githubusercontent.com/assets/1238659/19249034/2b2b4d7a-8ee8-11e6-9d9e-3fea9fafbbd9.png)

As an additional benefit, the user doesn't need to move the mouse as far when navigating back and forward in succession.

What do you think?